assignment1/fuel_indication.rb

```diff
@@ -0,0 +1,15 @@
+def fuel_indication n
+    if (n == 0)
+        puts 'Out of fuel'
+    elsif (1..10).include? n
+        puts 'Low, Please fill'
+    elsif (11..30).include? n
+        puts 'Good for now'
+    elsif (30..50).include? n
+        puts 'Almost Full'
+    elsif (n == 50)
```

Comment:
This code does not seem reachable, if you pass 50 then it will show "Almost Full" instead of "Full"
**Note**: I know there is a typo in the question ( Ex: 0 -> Out of fuel, 1-10 -> Low, Please fill, 11-30 -> Good for now, **31-50** -> Almost Full, > 50 -> Full) `31-50` should be `31-49`

Reply:
and also please try to handle floating points scenarios for eg. 30.5, 23.6, 45.9, etc,